docs/en/datasets/detect/index.md

```diff
@@ -170,6 +170,7 @@ Here is a list of the supported datasets and a brief description for each:
 - [African-wildlife](african-wildlife.md): A dataset featuring images of African wildlife, including buffalo, elephant, rhino, and zebras.
 - [Signature](signature.md): A dataset featuring images of various documents with annotated signatures, supporting document verification and fraud detection research.
 - [Medical-pills](medical-pills.md): A dataset featuring images of medical-pills, annotated for applications such as pharmaceutical quality assurance, pill sorting, and regulatory compliance.
+- [KITTI](kitti.md) New 🚀: A dataset featuring real-world driving scenes with stereo, LiDAR, and GPS/IMU data, used here for **2D object detection** tasks such as identifying cars, pedestrians, and cyclists in urban, rural, and highway environments.
```

Comment:
@glenn-jocher Do you think we should also add "3D detection", "depth estimation" after "2D object detection" now?
Or we can add after merging the code into main branch.